Add Proguard rules for models to prevent NullPointerException

I'm recieving `Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'char java.lang.String.charAt(int)' on a null object reference` inside `UnplashPickerAdapter.onBindViewHolder()` with `minifyEnabled true`.

Solution:

```
-keep class com.github.basshelal.unsplashpicker.data* {*;}
-keepclassmembers class com.github.basshelal.unsplashpicker.data* {
    <fields>;
    <init>();
    <methods>;
}
```

--- a/photopicker/proguard-rules.pro
+++ b/photopicker/proguard-rules.pro
@@ -25,3 +25,45 @@
 -dontwarn io.reactivex.functions.Function
 -dontwarn rx.internal.util.**
 -dontwarn sun.misc.Unsafe
+
+-keep class com.github.basshelal.unsplashpicker.data.UnsplashLinks {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.UnsplashLinks {
+ <fields>;
+ <init>();
+ <methods>;
+}
+
+-keep class com.github.basshelal.unsplashpicker.data.UnsplashPhoto {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.UnsplashPhoto {
+ <fields>;
+ <init>();
+ <methods>;
+}
+
+-keep class com.github.basshelal.unsplashpicker.data.NetworkEndpoints {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.NetworkEndpoints {
+ <fields>;
+ <init>();
+ <methods>;
+}
+
+-keep class com.github.basshelal.unsplashpicker.data.SearchResponse {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.SearchResponse {
+ <fields>;
+ <init>();
+ <methods>;
+}
+
+-keep class com.github.basshelal.unsplashpicker.data.UnsplashUrls {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.UnsplashUrls {
+ <fields>;
+ <init>();
+ <methods>;
+}
+
+-keep class com.github.basshelal.unsplashpicker.data.UnsplashUser {*;}
+-keepclassmembers class com.github.basshelal.unsplashpicker.data.UnsplashUser {
+ <fields>;
+ <init>();
+ <methods>;
+}

